osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSubScreen.cs

```diff
@@ -271,6 +271,7 @@ protected override void LoadComplete()
             UserMods.BindValueChanged(onUserModsChanged);
```

Comment:
Is this bind now removable, since all the code could be moved into `UpdateMods()`? Which also reveals that the `UpdateMods()` method probably should have a `client != null` check for the pre-join setup case.

Reply:
It can be, until #10707 I guess?

Reply:
I've removed the second call, but I don't think the client null check is required, as all calls to `UpdateMods` are scheduled, and the client is resolved. Unless I'm missing something?

Reply:
This was also not removable. It has a specific purpose. Reverting.